app/assets/stylesheets/frontend/views/_html-publications.scss

```diff
@@ -286,6 +286,71 @@
           }
         }
       }
+
+      table.financial-data {
+
+        .numeric {
+          text-align: right;
+          font-family: $NTA-Light-Tabular;
+        }
+
+        // make all elements inside thead look the same
+        // (td can sometimes be used to overcome complexity of multiple lines in thead)
+        // and make all of thead and tfoot stand out
+        thead,
+        tfoot {
+          background-color: #e1e8e8;
```

Comment:
FWIW you're right that that colour is already in use:

![screen shot 2016-10-28 at 15 23 03](https://cloud.githubusercontent.com/assets/1650875/19809743/80273cec-9d22-11e6-9370-a5b405cacb38.png)

Source: http://cssstats.com/stats?url=https%3A%2F%2Fgov.uk&ua=Browser%20Default

So if you like you can also just declare it at the top of the file using some throwaway name and move on. Or use the other grey. Your pick. 👍